src/main/java/org/mifos/pheevouchermanagementsystem/util/VoucherStatusEnum.java

```diff
@@ -4,7 +4,9 @@ public enum VoucherStatusEnum {
     INACTIVE("01"),
     ACTIVE("02"),
     CANCELLED("03"),
-    EXPIRED("04");
```

Comment:
you have to add string message for each Enum
 Eg:
INACTIVE("01" , "Inactive") ....;

the constructor will be like:



  VoucherStatusEnum(String code, String value) {